float64 new_x
float64 new_y
float64 new_yaw
---
float64 old_x
float64 old_y
float64 old_yaw